AsyncDisplayKit/ASCellNode.mm

```diff
@@ -119,8 +119,11 @@ - (void)layoutDidFinish
 
 - (void)__setNeedsLayout
 {
-  CGSize oldSize = self.calculatedSize;
+  CGSize oldSize = self.bounds.size;
```

Comment:
I'm not pretty sure if we have to do that necessarily as the layout pass in `__layout` needs to happen on the main thread. Changed to `_threadSafeBounds` though, as we hold the lock and it's a hot path.